### PR TITLE
Add actions in autocomplete for functions

### PR DESCRIPTION
**Pull request by caponetto**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: packages/cli/src/lib/bundler/config.ts [hunk 1]
```diff
@@ -179,7 +179,7 @@ export async function createConfig(
         child_process: false,
 
         /* new ignores */
-        path: false,
+        path: require.resolve('path-browserify'),
```
Comment (manstis) [inline packages/cli/src/lib/bundler/config.ts:182]: Is there any other workaround for this?

We want to minimise changes to Backstage itself.

Can it not be added to the overlay [here](https://github.com/manstis/backstage/blob/KOGITO-9338/packages/app/webpack.config.js)?

File: packages/cli/src/lib/bundler/transforms.ts [hunk 2]
```diff
@@ -169,6 +169,14 @@ export const transforms = (options: TransformOptions): Transforms => {
         filename: 'static/[name].[hash][ext][query]',
       },
     },
+    {
```
Comment (manstis) [inline packages/cli/src/lib/bundler/transforms.ts:172]: Likewise. Is there anyway to move this to the `app`'s `webpack.config`?

File: plugins/swf/src/api/api.ts [hunk 3]
```diff
@@ -19,10 +19,13 @@ import {
   SwfItem,
   SwfListResult,
 } from '@backstage/plugin-swf-common';
+import { OpenAPIV3 } from 'openapi-types';
 
 export interface SwfApi {
   getSwf(swfId: string): Promise<SwfItem>;
 
+  getActionsSchema(): Promise<OpenAPIV3.Document>;
```
Comment (manstis) [inline plugins/swf/src/api/api.ts]: Nit picking... can we keep related methods together?

You've put `getActionSchema()` in the middle of methods relating to retrieving SWF files.

You've put the method at the end of the implementation class.

File: plugins/swf/src/hooks/useServerlessWorkflowCombinedEditor.ts [hunk 4]
```diff
@@ -34,9 +34,15 @@ import { empty_definition } from '@backstage/plugin-swf-common';
...
-import { ServerlessWorkflowTextEditorChannelApi } from '@kie-tools/serverless-workflow-text-editor/dist/api';
 import { SwfLanguageServiceChannelApiImpl } from './SwfLanguageServiceChannelApiImpl';
+import { parseApiContent } from '@kie-tools/serverless-workflow-service-catalog/dist/channel';
+import { SwfCatalogSourceType } from '@kie-tools/serverless-workflow-service-catalog/dist/api/types';
+import { SwfServiceCatalogService } from '@kie-tools/serverless-workflow-service-catalog/dist/api';
+import { SwfCombinedEditorChannelApiImpl } from '@kie-tools/serverless-workflow-combined-editor/dist/impl';
+import { ServerlessWorkflowCombinedEditorChannelApi } from '@kie-tools/serverless-workflow-combined-editor/dist/api';
+
+// This filename should probably be returned by the backend
+const ACTIONS_OPEN_API_FILE = 'actions-openapi.json';
```
Comment (manstis) [inline plugins/swf/src/hooks/useServerlessWorkflowCombinedEditor.ts]: We use this file name [here](https://github.com/manstis/backstage/blob/7d98b25009d1246d01526bbcff64bd4c18e074d3/plugins/swf-backend/src/service/WorkflowService.ts#L55) too.

Could you please move this constant to [here](https://github.com/manstis/backstage/blob/7d98b25009d1246d01526bbcff64bd4c18e074d3/plugins/swf-common/src/constants.ts) and use it in both the place referenced above and your change here?

File: packages/cli/src/lib/bundler/config.ts (hunk 1 above)
Comment (caponetto) [reply packages/cli/src/lib/bundler/config.ts:182]: I've spent some time on this and I am still digging around to see if I come up with some other way, but, in the meantime, I guess we can merge this PR to move forward with other things. WDYT?

**Review by manstis — APPROVED** [review]

**Pull request merged**